test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -388,6 +390,13 @@ export class SampleAppIModelApp {
     let stageId: string;
     const defaultFrontstage = this.allowWrite ? EditFrontstage.stageId : ViewsFrontstage.stageId;
 
+    // Reset QuantityFormatter UnitsProvider with new iModelConnection
+    const schemaLocater = new ECSchemaRpcLocater(iModelConnection);
+    const context = new SchemaContext();
+    context.addLocater(schemaLocater);
+    IModelApp.quantityFormatter.unitsProvider = new SchemaUnitProvider(context);
+    await IModelApp.quantityFormatter.onInitialized();
```

Comment:
@christophermlawson I am getting crash here when opening existing briefcase/snapshots.  Is there an easy way to test to see if a units schema is available in the current connection before the unitsProvider is set?